docs/proposals/hpa/federated-hpa.md

```diff
@@ -31,7 +31,7 @@ this KEP.  Describe why the change is important and the benefits to users.
 1. Bring HPA from single cluster to multiple clusters.
 1. Compatible with the HPA related resources in the single cluster.
 1. Tolerate the disaster of member cluster or karmada control plane.
-1. It is better to integrate well with the scenarios such as workloads shifting and cloud burst.
+1. It is better to integrate well with the scenarios such as workloads shifting and cloud eruption.
```

Comment:
"shifting" and "eruption" are not the terms commonly used in cloud computing.
Consider "migration" and "scaling".
"cloud bursting" is a fixed term too, but it relates to on-prem infra.

Reply:
thanks for tips, I got it